backend/api/rooms.js

```diff
@@ -0,0 +1,28 @@
+const handler = require('../handler/handler');
+
+module.exports = (req, res) => {
+    // eslint-disable-next-line no-console
+    console.log('INFO: /rooms has been called.');
+    res.setHeader('Access-Control-Allow-Origin', '*');
```

Comment:
are there CORS issues? could we use the allowCors handler here?

Reply:
Just a error due to copy paste